revscoring/features/revision.py

```diff
@@ -72,41 +135,153 @@ def process_chars(revision_text):
 
 chars = Feature("revision.chars", process_chars,
                 returns=int, depends_on=[revision.text])
+"""
+Represents number of characters in the revsion.
+
+:Returns:
+    int
 
+:Example:
+    ..code-block:: python
+
+        >>> from revscoring.features import revision
+        >>> list(extractor.extract(655097130, [revision.chars]))
+        [24016]
+"""
 def process_markup_chars(revision_text):
     return sum(len(m.group(0)) for m in MARKUP_RE.finditer(revision_text))
 
 markup_chars = Feature("revision.markup_chars", process_markup_chars,
                        returns=int, depends_on=[revision.text])
+"""
+Represents number of markup characters in the revsion.
+
+:Returns:
+    int
+
+:Example:
+    ..code-block:: python
+
+        >>> from revscoring.features import revision
+        >>> list(extractor.extract(655097130, [revision.markup_chars]))
+        [704]
+"""
 
 proportion_of_markup_chars = markup_chars / modifiers.max(chars, 1)
+"""
+Represents ratio of markup characters compared to all characters in revsion.
 
+:Returns:
+    float
+
+:Example:
+    ..code-block:: python
+
+        >>> from revscoring.features import revision
+        >>> list(extractor.extract(655097130, [revision.proportion_of_markup_chars]))
+        [0.029313790806129246]
+"""
 def process_numeric_chars(revision_text):
     return sum(len(m.group(0)) for m in NUMERIC_RE.finditer(revision_text))
 
 numeric_chars = Feature("revision.numeric_chars", process_numeric_chars,
                         returns=int, depends_on=[revision.text])
+"""
+Represents number of numeric characters in the revsion.
+
+:Returns:
+    int
+
+:Example:
+    ..code-block:: python
+
+        >>> from revscoring.features import revision
+        >>> list(extractor.extract(655097130, [revision.numeric_chars]))
+        [203]
+"""
 
 proportion_of_numeric_chars = numeric_chars / modifiers.max(chars, 1)
+"""
+Represents ratio of numeric characters compared to all characters in revsion.
```

Comment:
"revision"  -- I'll stop noting this now.  Seems like a Ctrl-F "revsion" is in order.